share/spack/docker/leap-15.dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM opensuse/leap:15.2
+MAINTAINER Christian Goll <cgoll@suse.com>
+
+ENV DOCKERFILE_BASE=opensuse          \
+    DOCKERFILE_DISTRO=opensuse_leap   \
+    DOCKERFILE_DISTRO_VERSION=15.2    \
+    SPACK_ROOT_DOCKER=/opt/spack      \
```

Comment:
Here the recipe sets `SPACK_ROOT_DOCKER`
```suggestion
    SPACK_ROOT=/opt/spack      \
```
but later in the Dockerfile it's `SPACK_ROOT` to be referenced